meson/part/compiler_args/meson.build

```diff
@@ -109,6 +109,7 @@ endif
 
 extra_cppflags = extra_common_flags + [
 	'-fno-exceptions',
+	'-Wno-misleading-indentation',
```

Comment:
Because I was lazy to edit all the C++ code to add the missing braces around single command blocks ;-).

But seriously: we should probably update the code to never use `if (..) x()` but only  `if (..) { x() }`. But I think that can be done later in another PR.